apps/ledger-live-mobile/src/screens/Onboarding/steps/postWelcomeSelection.tsx

```diff
@@ -129,6 +138,10 @@ function PostWelcomeSelection({ route }: NavigationProps) {
     dark: images.dark[key],
   });
 
+  const setupLedgerImageSource = useFeature("staxWelcomeScreen")?.enabled
+    ? getSourceImageObj("setupLedgerStaxImg")
+    : getSourceImageObj("setupLedgerImg");
```

Comment:
should it be put in a `useMemo` ? I guess it depends if `PostWelcomeDiscoverCard` is memoized or not